tweb/src/test/java/io/fundrequest/platform/tweb/profile/ProfileControllerTest.java

```diff
@@ -54,9 +58,9 @@ void showProfile() throws Exception {
                 .andExpect(model().attribute("isVerifiedGithub", true))
                 .andExpect(model().attribute("isVerifiedStackOverflow", true))
                 .andExpect(model().attribute("refLink", "ref"))
-                .andExpect(model().attribute("refLinkTwitter", "ref"))
-                .andExpect(model().attribute("refLinkLinkedin", "ref"))
-                .andExpect(model().attribute("refLinkFacebook", "ref"));
+                .andExpect(model().attribute("refShareTwitter", "ref"))
```

Comment:
`when(messageService.getMessageByKey(..)).thenReturn(..)` should be added for the messageService calls

Reply:
@Simsteve7 this still needs to be done, the build is still failing